### PR TITLE
add basic functionality to display comments for a specific review

### PR DESCRIPTION
**Pull request by Ell534**
I will add more reactive styling later on, but for now the basic list is in place.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -22,4 +22,10 @@ const getCategories = () => {
   })
 }
 
-export { getReviews, getCategories, getReviewById };
+const getCommentsByReviewId = (review_id) => {
+  return api.get(`/reviews/${review_id}/comments`).then(({data}) => {
+    return data.comments;
```
Comment (lewispricey) [inline src/utils.js:27]: Great to see you're just returning the data you need in this function so theres no object diving in the comments component.

File: src/components/CommentsCards.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  console.log(commentsList);
+
+  if (commentsLoading) {
+    return <h4>Comments are loading, please wait...</h4>
+  }
+
+  return (
+    <>
+      <ul>
```
Comment (lewispricey) [inline src/components/CommentsCards.jsx]: Great use of the list here! The outer <> tag may be slightly redundant as the li wraps everything so you're only returning a single block.

File: src/components/CommentsCards.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  useEffect(() => {
+    setCommentsLoading(true)
+    getCommentsByReviewId(review_id).then((commentsFromApi) => {
+      setCommentsList(commentsFromApi);
+      setCommentsLoading(false);
+    });
+  }, []);
+
+  console.log(commentsList);
```
Comment (lewispricey) [inline src/components/CommentsCards.jsx]: Cheeky clog :)

File: src/components/Comments.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import CommentsCards from './CommentsCards';
+
+const Comments = ({review_id}) => {
+  return (
+    <>
+      <p>I am the comments, will be adding a post comment feature here</p>
```
Comment (lewispricey) [inline src/components/Comments.jsx:6]: This placeholder doesn't look to be required anymore :)

File: src/components/CommentsCards.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+  if (commentsLoading) {
```
Comment (lewispricey) [inline src/components/CommentsCards.jsx:16]: Love the seperate is loading for the comments so the user isnt waiting for both the article and comments to come back from the api before they can see anything

**Review by lewispricey — APPROVED** [review]
Great work on this ticket so far Elliot, I've left a few comments on the code itself but overall its awesome!

I would recommend adding in a check for if there are no comments to display a message to the user to say 'no comments for this article/review' but other than that you're good to go.

**Comment by Ell534** [comment]
Added in the check for no comments, with a relevant message to be displayed

**Pull request merged**